### PR TITLE
Add multiple choice and diagram to "The Box"

### PR DESCRIPTION
**Pull request by daverosoff**
Addresses https://github.com/TeamBasedInquiryLearning/library/issues/51.

### CODE REVIEW

**Comment by StevenClontz** [comment]
We'll need a non-SVG version for print. The least annoying solution is to make a PNG that can be used in both HTML and PDF formats (but keep the source SVG here for the future).

File: calculus/source/03-AD/08.ptx [hunk 1]
```diff
@@ -12,30 +12,96 @@
...
-            <p>Draw a diagram illustrating how the box is created.</p>
+            <statement>
+              <p>Draw a diagram illustrating how the box is created. Label the lengths of segments in the diagram and write <m>x</m> for the
+              sides of the smaller squares to be removed from the 8 by 8 square.</p>
+            </statement>
+          <solution>
+            <image source="thebox-1">
+              <shortdescription>Square of side length 8 with square corners of side length <m>x</m> removed</shortdescription>
+            </image>
+          </solution>
```
Comment (StevenClontz) [inline calculus/source/03-AD/08.ptx:36]: For the purpose of a 4S in-class activity, I think providing four choices of diagrams and letting students choose which is most appropriate is better scaffolded, especially this early in a section.

**Comment by daverosoff** [comment]
More like this? I think the pictures are OK, not sure the best way to give them labels A through D.

**Comment by StevenClontz** [comment]
> More like this? I think the pictures are OK, not sure the best way to give them labels A through D.

Looks good - I'd either manually insert the letters in the images, or use a `<ol marker="A." cols="2"/>`.

**Comment by daverosoff** [comment]
This still needs the PNG images which I have to make on the computer that has Inkscape.

**Comment by daverosoff** [comment]
Closed per discussion at #124.

**Pull request closed without merging**